package.json

```diff
@@ -22,7 +22,10 @@
     "build:scripts": "vite build",
     "build:types": "tsc --project tsconfig.build.json --emitDeclarationOnly",
     "lint": "eslint --ext .tx,.ts .",
-    "version:get": "node -p \"require('./package.json').version\""
+    "version:get": "node -p \"require('./package.json').version\"",
```

Comment:
This isn't necessary

Reply:
Is there a reason to remove it? It defined was on our package.json before this update

Reply:
I'll leave this script, since it's currently being used on our release job. Changed the other ones!